frontend/src/components/Profile.vue

```diff
@@ -0,0 +1,192 @@
+            <select name="Gender" id="gender" required>
+              <option value="Gender" disabled selected>Gender*</option>
+              <option value="Male">Male</option>
+              <option value="Female">Female</option>
+              <option value="Other">Other</option>
+            </select>
+            <span class="container__select-icon"
+              ><i class="zmdi zmdi-chevron-down"></i
+            ></span>
+          </div>
+        </div>
+
+        <div class="container__group">
+          <input
+            type="text"
+            placeholder="Nationality"
+            name="nationality"
+            id="nationality"
+          />
+        </div>
+
+        <div class="container__group">
+          <div class="container__select">
+            <select name="languages" id="languages">
+              <option value="Language" disabled selected>Languages</option>
+              <option value="French">English</option>
+              <option value="Dutch">Dutch</option>
+              <option value="English">French</option>
+              <option value="Russian">Russian</option>
+              <option value="Spanish">Spanish</option>
+              <option value="Arabic">Arabic</option>
+              <option value="Hindu">Hindu</option>
+              <option value="Chinese">Chinese</option>
+              <option value="Italian">Italian</option>
+            </select>
+            <span class="container__select-icon"
+              ><i class="zmdi zmdi-chevron-down"></i
+            ></span>
+          </div>
+        </div>
+        <div class="container__group">
+          <div class="container__select">
+            <select name="skills" id="skills">
+              <option value="Skills" disabled selected>Skills</option>
+              <option value="Translator">Legal assistance</option>
+              <option value="Translator">Translations</option>
+              <option value="Host">Mental health</option>
+              <option value="assistant">Host family</option>
+              <option value="assistant">Education services</option>
+              <option value="assistant">Language classes</option>
+              <option value="assistant">Social assistance</option>
+            </select>
+            <span class="container__select-icon"
+              ><i class="zmdi zmdi-chevron-down"></i
+            ></span>
+          </div>
+        </div>
+
+        <textarea
+          class="container__description"
+          placeholder="Description"
+        ></textarea>
+
+        <div class="container__modify">
+          <div class="container__modify-delete">
+            <button class="container__modify-button">Delete My Profile</button>
+          </div>
+          <div class="container__modify-edit">
+            <input
+              type="button"
+              value="EDIT"
+              class="submit"
+              name="edit"
+              id="edit"
+            />
+            <input
```

Comment:
maybe try creating a separate div for this button..now the text does not stay in the center on smaller screen width..

Reply:
Ok Daniel. I will do 👍